src/App.css

```diff
@@ -28,6 +42,66 @@
   color: #61dafb;
 }
 
+h1 {
+  font-family: 'Montez', cursive;
+  font-size: 4.5rem;
+  color: #D4AF37;
+  border: 2px solid #000;
+}
+
+h2 {
+  font-family: 'Montez', cursive;
+  /* font-family: 'Rock Salt', cursive; */
+  font-size: 2rem;
+  color: rgb(7, 88, 7);
+}
+
+form,
+label,
+select,
+button {
+  padding: 10px 10px;
+  font-size: 1.5rem;
+}
+
+select {
+  /* color: #D4AF37; */
+  border: 1px solid #000;
+  border-radius: 5%;
+  
+}
+
+button {
+  display: inline-block;
+  margin-left: 10px;
+  /* color:#D4AF37; */
+  font-weight: 500;
+  border: 1px solid #000;
+  border-radius: 5%;
+}
+
+p {
+  font-family: 'Rock Salt', cursive;
+  font-size: 2rem;
+  font-weight: 700;
+  display: inline-block;
+  padding: 2%;
+  border-bottom: 3px solid #000;
+}
+
+
+/* footer {
+  min-height:50px;
+  margin-top: auto; 
+} */
+
+footer {
+  position: absolute;
```

Comment:
Very innovating approach to keeping the footer at the bottom. Consider adding position: relative + min-height: 100vh to the .app class to ensure the footer is staying at the bottom.